SU2_CFD/src/drivers/CDriver.cpp

```diff
@@ -1244,6 +1255,22 @@ void CDriver::Solver_Preprocessing(CConfig* config, CGeometry** geometry, CSolve
   
 }
 
+void CDriver::MeshSolver_Preprocessing(CSolver ***solver, CGeometry **geometry, CConfig *config) {
```

Comment:
Hm can we move that to the solver preprocessing routine ?

Reply:
Yep, I just did. I'm sure I had a reason when I implemented the separate routine, but I agree it doesn't make sense as it stands now.